transport/config/config.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import "os"
+
+const (
+	// AuthHeader - Name of the HTTP header to use for authentication and
+	// authorization.
+	AuthHeader = "Authorization"
+
+	// AuthHeaderPrefix - Prefix expected for the HTTP auth header value.
+	AuthHeaderPrefix = "Bearer "
```

Comment:
Very minor point and I know it's probably always been like this but it'd be friendlier if the prefix was just "Bearer" and the concatenation logic added in the space?

Reply:
I concur with you there @codingconcepts. That was a bit of a dick move on my part wasn't it :p